server/src/main/java/org/apache/druid/server/coordinator/DruidCoordinatorConfig.java

```diff
@@ -47,9 +45,15 @@ public abstract class DruidCoordinatorConfig
   @Default("false")
   public abstract boolean isKillUnusedSegmentsEnabled();
 
+  /**
+   * @return the coordinator's kill period duration. If the config isn't explicitly set,
+   * it defaults to the coordinator's indexing period.
```

Comment:
"coordinator's kill period" is very misleading. In fact, we should even rename the method but maybe not needed in this PR. There is another open PR which rewires all of these configs, where this will be addressed: #15705 

```suggestion
   * @return The period at which the coordinator cleans up unused segments. If the config isn't explicitly set,
   * it defaults to the coordinator's indexing period.
```